README.md

```diff
@@ -12,6 +12,17 @@ Monorepo to hold all shareable linting and formatting configurations.
 
 When using both ESLint and Prettier in a project, it's nice to have files automatically have ESLint errors fixed on save as well as have non JavaScript/TypeScript files formatted on save. This section explains how to set that up in an editor.
 
+### Versions Bump
+Patch releases: 
+  - Bump dependencies to patch or minor releases
+  - Fixs not impacting the configs or adding new rules
+Minor releases: 
+  - Adding exceptions to rules (ex: disable a default rule)
+  - Changin Minor rules (Nothing that will reformat code by default)
```

Comment:
Change any linting rule. Also `Changin` is misspelled.

Reply:
yeah my bad i wrote this in like 30 seconds and I have Bad Grammar reputation 😆